resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -429,13 +440,12 @@ class TextCursor
     [DscProperty()] [nullable[bool]] $IndicatorStatus
     [DscProperty()] [int] $IndicatorSize
     [DscProperty()] [int] $IndicatorColor
-    [DscProperty()] [int] $Thickness
+    [DscProperty()] [int] $KeyboardShortcutStatus
 
     static hidden [string] $IndicatorStatusProperty = 'Configuration'
     static hidden [string] $IndicatorStatusValue = 'cursorindicator'
-    static hidden [string] $IndicatorSizeProperty = 'IndicatorType'
-    static hidden [string] $IndicatorColorProperty = 'IndicatorColor'
-    static hidden [string] $ThicknessProperty = 'CaretWidth'
+    static hidden [string] $IndicatorColorProperty = 'FilterType'
+    static hidden [string] $KeyboardShortcutStatusProperty = 'HotkeyEnabled'

```

Comment:
Did you mean to edit these?

Reply:
No. I've been trying to get too much done here, and in the winget-pkgs repo, causing me to fail at both.